src/ansys/additive/core/additive.py

```diff
@@ -42,57 +42,42 @@
 from ansys.additive.core.download import download_file
 from ansys.additive.core.material import AdditiveMaterial
 from ansys.additive.core.material_tuning import MaterialTuningInput, MaterialTuningSummary
-from ansys.additive.core.microstructure import MicrostructureSummary
+from ansys.additive.core.microstructure import MicrostructureInput, MicrostructureSummary
 import ansys.additive.core.misc as misc
-from ansys.additive.core.porosity import PorositySummary
+from ansys.additive.core.porosity import PorosityInput, PorositySummary
 from ansys.additive.core.progress_logger import ProgressLogger
 from ansys.additive.core.server_connection import ServerConnection
 from ansys.additive.core.simulation import SimulationError
-from ansys.additive.core.single_bead import SingleBeadSummary
+from ansys.additive.core.single_bead import SingleBeadInput, SingleBeadSummary
 from ansys.additive.core.thermal_history import ThermalHistoryInput, ThermalHistorySummary
 
 
 class Additive:
-    """Provides the client interface to the Additive service.
-
-    You can connect to the Additive server using one of the following
-    methods. The methods are listed in order of precedence.
-        1. If ``channel`` is provided, use it to connect to the server.
-        2. If ``host``, and optionally ``port``, are provided, connect to the server at ``host:port``.
-        3. If running in a :class:`PyPIM <ansys.platform.instancemanagement.pypim>`-
-        enabled cloud environment, launch and connect to an ``additive`` service.
-        4. Use the value of the ``ANSYS_ADDITIVE_ADDRESS`` environment variable if it is defined.
-        The value uses the format ``host:port``.
-        5. Attempt to start the server on localhost and connect to it. For this to work,
-        the Additive portion of the Ansys Structures package from the Ansys unified installation
-        must be installed.
-
+    """Provides the client interface to one or more Additive services.
 
     Parameters
     ----------
     server_connections: list[str, grpc.Channel], None
         List of connection definitions for servers. The list may be a combination of strings and
-        connected ``grpc.Channel``s. Strings use the format ``host:port`` to specify
-        the server IPv4 address.
-    channel: grpc.Channel, None
-        gRPC channel connection to use for communicating with the server. If None, a connection will be
+        connected :class:`grpc.Channel <grpc.Channel>` objects. Strings use the format
+        ``host:port`` to specify the server IPv4 address.
+    channel: grpc.Channel, default: None
+        gRPC channel connection to use for communicating with the server. If ``None``, a connection will be
         established using the host and port parameters. Ignored if ``server_channels``
         is not ``None``.
```

Comment:
```suggestion
        gRPC channel connection to use for communicating with the server. If the
        default ``None`` is used, a connection is established using the host and port
        parameters. This parameter is ignored if the ``server_channels`` parameter
        is other than ``None``.
```
After the brief description, use complete sentences and avoid future tense. In the last
line, did you mean ``server_channels``? I just don't see this parameter and wanted to
be sure it was what you wanted.